Wurst Client for MC 1.9.X/src/tk/wurst_client/mods/AutoMineMod.java

```diff
@@ -33,9 +33,11 @@ public void onUpdate()
 			|| mc.objectMouseOver.getBlockPos() == null)
 			return;
 		if(Block.getIdFromBlock(mc.theWorld.getBlockState(
-			mc.objectMouseOver.getBlockPos()).getBlock()) != 0)
-			mc.gameSettings.keyBindAttack.pressed = true;
-		else
+			mc.objectMouseOver.getBlockPos()).getBlock()) != 0){
+			if(wurst.mods.autoToolMod.isActive())
+				AutoToolMod.setSlot(mc.objectMouseOver.getBlockPos());
+				mc.gameSettings.keyBindAttack.pressed = true;
```

Comment:
I installed Wurst Client in eclipse but...

```java
Exception in thread "main" joptsimple.MissingRequiredOptionException: Missing required option(s) ['userProperties', 'accessToken', 'version']
	at joptsimple.OptionParser.ensureRequiredOptions(OptionParser.java:375)
	at joptsimple.OptionParser.parse(OptionParser.java:365)
	at net.minecraft.client.main.Main.main(Main.java:51)
```
Please help me, how can I fix it?